CHANGELOG.next.asciidoc

```diff
@@ -12,6 +12,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Update to Golang 1.12.1. {pull}11330[11330]
 - Update to Golang 1.12.4. {pull}11782[11782]
+- Add configurable bulk_flush_frequency in kafka output. {pull}12254[12254]
```

Comment:
This should be moved to the "Added" section.

Reply:
I have moved this and approved the PR, I will merge once builds are green again.